authors/apps/articles/tests/test_highlight.py

```diff
@@ -0,0 +1,71 @@
+from .base_test import TestBaseCase
+
+from rest_framework import status
+
+
+class TestHighlight(TestBaseCase):
+    """Class to test the highlight functionality"""
+
+    def highlight(self):
+        """Function to try all highlight possibilities"""
+        self.authorize()
+        slug = self.create_article()
+        response1 = self.client.post(self.highlight_url(
+            slug), self.highlight_data, format='json')
+        response2 = self.client.post(self.highlight_url(
+            slug), self.highlight_out_of_range, format='json')
+        response3 = self.client.post(self.highlight_url(
+            slug), self.start_index_greater_than_end_index, format='json')
+        response4 = self.client.post(self.highlight_url(
+            slug), self.highlight_index_not_integer, format='json')
+        response5 = self.client.post(self.highlight_url(
+            slug), self.missing_highlight_field, format='json')
+        response6 = self.client.post(self.highlight_url(
+            "invalid slug"), self.highlight_data, format='json')
+        responses = [response1, response2, response3, response4,
+                     response5, response6]
+        return responses
+
+    def base_error_messages(self, response, message):
+        self.assertIn(message.encode(), response.content)
```

Comment:
I like the use of the function as it is reusable. 
Keep up the good use of reusability.